.ci/Jenkinsfile

```diff
@@ -220,6 +220,66 @@ pipeline {
         }
       }
     }
+    stage('Stack Release Tag') {
+      options {
+        skipDefaultCheckout()
+      }
+      when {
+        beforeAgent true
+        allOf {
+          branch 'main'
+          expression { return params.release }
+        }
+      }
+      agent { label 'ubuntu-18 && immutable' }
+      stages {
+        stage('Notify') {
+          options { skipDefaultCheckout() }
+          steps {
+            notifyStatus(slackStatus: 'warning', subject: "[${env.REPO}] Stack Release ready to be pushed",
+                         body: "Please go to (<${env.BUILD_URL}input|here>) to approve or reject within 12 hours.")
+          }
+        }
+        stage('Stack Release CI') {
+          options { skipDefaultCheckout() }
+          input {
+            message 'Should we tag the current version as a stack release?'
+            ok 'Yes, we should.'
+          }
+          environment {
```

Comment:
Will this `input` support for a `cancel`? If so, what's gonna happen?

Reply:
I switched to a prompt pattern instead of using an input pattern.